.github/workflows/code.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           - suite: "base"
             build-args: ""
           - suite: "try-runtime"
-            build-args: "--features try-runtime"
+            build-args: "--features try-runtime,with-tracing"
```

Comment:
I don't like this - enabling something like this via CI would result in a narrow case of when this feature is on. I think we shouldn't enable it just for CI - like if you build and run node for try-runtime locally you'd expect it to work the same way as CI.

So, at the very least, I'd suggest providing an easier route flag for the end user to enable this feature - if we need it.

Reply:
Got your point, removed from CI - https://github.com/humanode-network/humanode/pull/1325/commits/1e92f329135f06392a60021a0c5640936669c91e